utils/getDashboardData.mjs

```diff
@@ -0,0 +1,54 @@
+/**
+ * Get dashboard data
+ * @param db – Sequelize/Postgress database
+ * @returns object { sections, ... }
+ */
+const getDashboardData = async (db, userId) => {
```

Comment:
Having a function like this in `utils` is an anti-pattern. This is because this function is "tightly-coupled", i.e. "very dependent" on the specific nature of your app and the specific contents of `db`. 

Typically we want to put functions in `utils` that are more generic, e.g. getting a specific date format, or performing a more generic calculation that isn't app-dependent.

Let's put `getDashboardData` in the dashboards controller file instead?